guide/announcements.md

```diff
@@ -4,6 +4,105 @@ Welcome to the TDS Announcement Board! Check back here regularly for the latest
 
 <hr/>
 
+## March 2019
+
+### New Core Components ✨
+
+**Benefit**
+Want a clear, concise and design-approved way to tell your users how great something is? Benefit is for you! Take a look over at the [Benefit docs (Heading)](https://tds.telus.com/components/index.html#!/BenefitWithHeading) and [Benefit docs (No Heading)](https://tds.telus.com/components/index.html#!/BenefitNoHeading)
+
+**Video** and **WebVideo**
+TDS now offers a consistent video experience! Have a video hosted on TELUS servers? Use the Video component to take advantage of a full video player experience! Want to use a YouTube video? We have a component for that too! WebVideo will insert YouTube videos into your page along with using a TELUS branded splash screen. Take a look at the [Video docs](https://tds.telus.com/components/index.html#!/Video) and [WebVideo docs](https://tds.telus.com/components/index.html#!/WebVideo)
+
+### Features 🎁
+
+**DecorativeIcon 1.1.0**
```

Comment:
Do we want to use consistent naming when referring to components? Either "DecorativeIcon 1.1.0" or "@tds/core-decorative-icon@1.1.0"